### PR TITLE
Livefyre require ify

### PR DESCRIPTION
**Pull request by joseph-ross-work**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "streamhub-map",
-  "version": "1.2.1",
+  "version": "1.2.2",
```
Comment (gobengo) [inline bower.json]: 1.3 plz

File: examples/howsfseessf-built/index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" type="text/css" href="../../lib/streamhub-sdk/src/css/style.css">
```
Comment (gobengo) [inline examples/howsfseessf-built/index.html]: This shouldn't be needed since CSS is being inlined right?

File: package.json [hunk 3]
```diff
@@ -4,15 +4,19 @@
   "author": {
     "name": "cheung31"
   },
-  "version": "1.2.1",
+  "version": "1.2.2",
```
Comment (gobengo) [inline package.json]: 1.3

File: examples/howsfseessf-built/index.html (hunk 2 above)
Comment (joseph-ross-work) [inline examples/howsfseessf-built/index.html]: Are we suppose to be inlining the sdk css with every module? The reason for pull this down is that the base styling for the modal comes from the sdk and not the css for the map.

**Pull request merged**